### ISSUE TITLE
Referenced sniff "SlevomatCodingStandard.*" does not exist after upgrading to v8.24

### ISSUE THREAD

**Issue by arodu** [slevomat/coding-standard#1799]
Hi,

After updating to `slevomat/coding-standard` **v8.24**, I started getting multiple errors when running `phpcs` or `phpcbf`.

Example output:

```
ERROR: Referenced sniff "SlevomatCodingStandard.Arrays.TrailingArrayComma" does not exist.
ERROR: Referenced sniff "SlevomatCodingStandard.Functions.RequireTrailingCommaInDeclaration" does not exist.
ERROR: Referenced sniff "SlevomatCodingStandard.Functions.RequireTrailingCommaInClosureUse" does not exist.
ERROR: Referenced sniff "SlevomatCodingStandard.Functions.RequireTrailingCommaInCall" does not exist.
...
ERROR: Referenced sniff "SlevomatCodingStandard.ControlStructures.RequireShortTernaryOperator" does not exist.
```

### Environment

* **PHP:** 8.2.29
* **Composer:** 2.8.12
* **PHP_CodeSniffer:** 4.0.0
* **slevomat/coding-standard:** 8.24
* **Other related libs:** `cakephp/cakephp-codesniffer` (in my case)

### Steps to reproduce

1. Install `slevomat/coding-standard` v8.24 via Composer.
2. Run `vendor/bin/phpcs -i` → `SlevomatCodingStandard` does appear.
3. Run `composer cs-check` (or directly `vendor/bin/phpcs ./src ./tests`).
4. Errors above are thrown.

### Expected behavior

Referenced sniffs should be available and phpcs should run normally (as it does in v8.23).

### Actual behavior

All references to SlevomatCodingStandard.* sniffs are reported as “does not exist” in v8.24.

---------

It looks like some sniffs may have been renamed, moved, or removed in `v8.24`, but the references are still present in rulesets (e.g. CakePHP Codesniffer). Could you confirm if this is intentional (BC break) or an issue with the release?

Thanks in advance! 🙏
